downloads/github-actions-cheat-sheet.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: cheat-sheet
+redirect_to: false
+title: GitHub Actions Cheat Sheet
+byline: GitHub Actions give you the flexibility to build automated software development lifecycle workflows. You can write individual tasks, called actions, and combine them to create  custom workflows in your repository. GitHub Actions are automated processes allowing you to build, test, package, release, or deploy any code project on GitHub, but you can also use them to automate any step of your workflow&#58; merging pull requests, assigning labels, triaging issues to name a few.
```

Comment:
```suggestion
byline: GitHub Actions give you the flexibility to build automated software development lifecycle workflows. You can write individual tasks, called actions, and combine them to create custom workflows in your repository. GitHub Actions are automated processes allowing you to build, test, package, release, or deploy any code project on GitHub, but you can also use them to automate any step of your workflow&#58; merging pull requests, assigning labels, triaging issues to name a few.
```

Removed extra whitespace, but what about the `workflow&#58`, I think the `;` is already there. Should we take it out?

Reply:
":" is a Jekyll delelimiter so I had to use the HTML code